tests/taskTests/s3Download/s3Download-test.ts

```diff
@@ -35,14 +35,29 @@ describe('S3 Download', () => {
 
     const listObjectsResponse = {
         promise: function() {
-            return { NextMarker: undefined, Contents: undefined }
+            return { NextContinuationToke: undefined, Contents: undefined }
```

Comment:
looks like a typo (similar for line 43, etc)

```suggestion
            return { NextContinuationToken: undefined, Contents: undefined }
```

Reply:
Fixed, these tests really need to be cleaned up as I wrote them when I didn't know typescript or vsts at all